package.json

```diff
@@ -10,7 +10,7 @@
   "dependencies": {
     "@buffetjs/core": "^3.3.8",
     "@buffetjs/styles": "^3.3.8",
-    "@elastic/elasticsearch": "~7.13.0",
+    "@elastic/elasticsearch": "~8.0.0",
```

Comment:
It was set to be on `7.13.0` because for latest versions it was incompatible with Elasticsearch distributions from AWS. What distribution did you try on?

> More about that [here](https://github.com/elastic/elasticsearch-js/issues/1519)

Reply:
It's self-hosted, I guess maybe just release it under some beta version.